save-orchestrator/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ tasks.withType<Test> {
 
 dependencies {
     api(projects.saveCloudCommon)
+    api(projects.testUtils)
```

Comment:
As far as I understood we need to use api here because it is a replace of compile keyword (https://stackoverflow.com/questions/44413952/gradle-implementation-vs-api-configuration).

Reply:
`api` is needed, when there some project consumes this project as a dependency and you want a transitive dependency to be added to that project too. If you only need to use dependency in this project, without marking it for other consumers, `implementation` should be used. Basically, there is no observable difference here, just `implementation` is more common in such cases. Not sure, why we added save-cloud-common as `api`

Reply:
Ok, got it. Thanks for proper answer.